### PR TITLE
Update validation for API Priority and Fairness

### PR DESCRIPTION
**Pull request by MikeSpreitzer**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespace from that line:
>
> /kind api-change
> /kind bug
> /kind cleanup
> /kind deprecation
> /kind design
> /kind documentation
> /kind failing-test

/kind feature

> /kind flake

**What this PR does / why we need it**:
This PR fixes oversights and adds validation that rejects writes
of wrong Spec values for the four mandatory objects.  The added validation
makes it difficult (but not impossible in conceivable mixed-release scenarios)
for admins to successfully do an operation that will soon be undone.

**Which issue(s) this PR fixes**:

**Special notes for your reviewer**:
The feature is still being introduced.
This PR is a part of what is in #85319 

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs
[KEP] https://github.com/kubernetes/enhancements/blob/master/keps/sig-api-machinery/20190228-priority-and-fairness.md
```

### CODE REVIEW

**Comment by MikeSpreitzer** [comment]
/cc @lavalamp 
/cc @deads2k 
/cc @yue9944882 
/cc @aaron-prindle 
@mars1024

**Comment by MikeSpreitzer** [comment]
@MikeSpreitzer

**Comment by MikeSpreitzer** [comment]
/cc @jktomer 
@bjrara

File: pkg/apis/flowcontrol/internalbootstrap/default-internal.go [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+// value is the `*FlowSchema`.  Nobody should mutate anything
+// reachable from this map.
+var MandatoryFlowSchemas = internalizeFSes(bootstrap.MandatoryFlowSchemas)
+
+// MandatoryPriorityLevels holds the untyped renditions of the
+// mandatory priority level configuration objects.  In this map the
+// key is the object's name and the value is the
+// `*PriorityLevelConfiguration`.  Nobody should mutate anything
+// reachable from this map.
+var MandatoryPriorityLevels = internalizePLs(bootstrap.MandatoryPriorityLevelConfigurations)
```
Comment (yue9944882) [inline pkg/apis/flowcontrol/internalbootstrap/default-internal.go]: nit: MandatoryPriorityLevelConfigurations

**Comment by MikeSpreitzer** [comment]
The force-push to 796063e697f fixed a bazel oversight.

**Comment by MikeSpreitzer** [comment]
The force-push to 4d88acee51d59d1b4179751a42cd9e1139f25764 fixed some editing oversights.

**Comment by lavalamp** [comment]
/lgtm
/approve

Thanks!

**Pull request merged**